pkg/subnet/local.go

```diff
@@ -155,12 +155,20 @@ func (d *LocalDeployer) doDeploy(chain string, chainGenesis []byte, genesisPath
 
 	if !networkBooted {
 		if err := d.startNetwork(ctx, cli, avalancheGoBinPath, runDir); err != nil {
+			secondErr := d.removeInstalledPlugin(chainVMID)
```

Comment:
we can avoid this one by moving installPlugin just before calling CreateBlockchains

Reply:
moved it